src/test/java/org/javarosa/core/model/utils/test/QuestionPreloaderTest.java

```diff
@@ -15,13 +16,30 @@
 import java.io.IOException;
 import org.javarosa.core.test.Scenario;
 import org.javarosa.xform.parse.XFormParser;
-import org.junit.Ignore;
 import org.junit.Test;
 
 public class QuestionPreloaderTest {
-    @Ignore("Not supported")
     @Test
-    public void preloader_preloadsAttributes() throws IOException, XFormParser.ParseException {
+    public void preloader_preloadsElements() throws IOException, XFormParser.ParseException {
+        Scenario scenario = Scenario.init("Preload attribute", html(
+            head(
+                title("Preload element"),
+                model(
+                    mainInstance(t("data id=\"preload-attribute\"",
+                        t("element")
+                    )),
+                    bind("/data/element").preload("uid")
+                )
+            ),
+            body(
+                input("/data/element")
+            )));
+
+        assertThat(scenario.answerOf("/data/element").getDisplayText(), startsWith("uuid:"));
+    }
+
+    @Test
+    public void preloader_doesNotpreloadAttributes() throws IOException, XFormParser.ParseException {
```

Comment:
I think it'd be good to have a comment here highlighting that we might want this to **not** be the case one day. Makes it clear this test is documentation rather than intention.